testing/fuchsia/run_tests.py

```diff
@@ -102,18 +104,31 @@ def bundled_test_runner_of(target_id: str) -> BundledTestRunner:
           tests
```

Comment:
nit: 'variant' not in test (in previous line)

Reply:
Done.